android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -91,7 +91,7 @@ public void onBindViewHolder(SessionViewHolder holder, int position) {
                 + ISO8601Date.getTimeZoneDateString(ISO8601Date.getDateObject(session.getStartTime())).split(",")[1];
 
         holder.sessionTitle.setText(session.getTitle());
-        holder.sessionAbstract.setText(session.getSummary());
+        //holder.sessionAbstract.setText(session.getSummary());
```

Comment:
@arp95 Why did you remove this?
We need the abstract for each session in the card.